src/app/builder/builder.component.html

```diff
@@ -122,32 +132,78 @@
       <h4>Copy and Use Your Embed Code</h4>
 
       <p>Copy the code for one of the following players, and place it somewhere on your webpage.</p>
+      <section class="builder-step" id="step-4a" *ngIf="props">
+          <h4>
+            <input type="checkbox" [checked]="playPlaylist" (change)="togglePlayPlaylist()">
+            Or: Make It A Playlist
```

Comment:
Since you can't customize the player unless you have a specific episode selected, shouldn't those options be grouped? I feel like putting the playlist option between the specific episode selection and the customization would separate two sets of controls that should go together. 

What if the playlist option was the first thing selected above the Always Play Most Recent Episode? This is me coming from the perspective of I want to make a playlist player as one of my initial requirements. I don't know if that's how a typical user approaches this though.

The "Or:" on "Or: Make It A Playlist" makes me think it is mutually exclusive from the other options of "Always Play Most Recent Episode" and "Or: Select Episode". I would drop the "Or: " from "Make It A Playlist" even if leaving it where it is.